src/components/action/features/FeaturesSetings.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import Select from 'react-select';
+
+const options = [
```

Comment:
We will read features list from the server, it will return the list of human friendly list already. So, let's move this info into 'features' reducer and use same value for both 'value' and 'label'. 
PS: you can use lodash (_.map) to convert strings into options object as react-select expects